repository/src/main/java/org/alfresco/schedule/ScheduledJobLockExecuter.java

```diff
@@ -83,52 +86,42 @@ public ScheduledJobLockExecuter(JobLockService jobLockService, String name, Abst
      * @throws JobExecutionException        thrown if the job fails to execute
      */
     public void execute(JobExecutionContext jobContext) throws JobExecutionException
-    {
+    {
+        LockCallback lockCallback = new LockCallback();
         try
         {
-            if (logger.isDebugEnabled())
-            {
-                logger.debug(String.format("   Job %s started.", lockQName.getLocalName()));
-            }
-            refreshLock();
+            LOGGER.debug(String.format("   Job %s started.", lockQName.getLocalName()));

```

Comment:
You have to use the slf4j token insertion syntax otherwise you don't get any benefit from this.
```suggestion
            LOGGER.debug("   Job {} started.", lockQName.getLocalName());
```
There's also a "fluent" API for slf4j that avoids premature evaluation of getLocalName (which is presumably a tiny saving for this particular method). I think it's pretty hard to read though:
```
LOGGER.atDebug().addArgument(() -> lockQName.getLocalName()).log("   Job {} started.");
```

Reply:
Using slf4j token insertion.